### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by ajn404** [satnaing/astro-paper#252] [Show and tell]
Based on this template, this year I have launched my own astro blog. While documenting my blog, I have been concurrently learning and have made significant progress. I am grateful for the **astro paper** and the contributions of the original author. The next phase might mark the final chapter of my front-end career, as I am transitioning to become a supermarket cashier. I extend my gratitude for all the technologies I have utilized over the past three years.

**Comment by pahaz** [comment]
I also wanted to say thanks as well ❤️! I got my blog up and running on my own domain in just 10 minutes 🚀.

It would be awesome to see projects implemented with this template. 
My result: https://pahaz.8iq.dev/ 
Repo: https://github.com/pahaz/pahaz.8iq.dev

Big thanks!

**Comment by satnaing** [comment]
Thank you both for the kind words and for sharing your experiences with AstroPaper! ✌🏼
It’s always interesting to see what others have built with their own versions of the blog using AstroPaper.

@ajn404, I’m glad to hear AstroPaper has helped you on your journey. Thanks for sharing this, and best of luck with your next steps!

@pahaz , great to see your blog up and running so quickly! Thanks for sharing your repo with the community_ it is a fantastic example of how the template can be used.

**Comment by eerison** [comment]
I also want to say my **THANK SO YOU MUCH** for this amazing project ❤️

**Comment by vsme** [comment]
Thank you very much for this project, I developed my blog based on it
My result: https://bgo.me/
Repo: https://github.com/vsme/astro-obsidian-blog
